dist/js/joomla-a11y-checker.umd.js

```diff
@@ -2505,7 +2505,7 @@
                     'beforebegin',
                     this.annotate(
                         Lang._('WARNING'),
-                        `${Lang.sprintf('CONTRAST_WARNING_MESSAGE', nodetext)} <hr aria-hidden="true"> ${Lang._('CONTRAST_WARNING_MESSAGE_INFO')}`
+                        `${Lang.sprintf('CONTRAST_WARNING_MESSAGE')} <hr aria-hidden="true"> ${Lang._('CONTRAST_WARNING_MESSAGE_INFO', nodetext)}`, true
```

Comment:
this will not work :wink: 
you need remove `sprintf` from first `Lang`, and add to second `Lang`:

```
${Lang._('CONTRAST_WARNING_MESSAGE')} <hr aria-hidden="true"> ${Lang.sprintf('CONTRAST_WARNING_MESSAGE_INFO', nodetext)}
```

Reply:
dont know how I missed that- thanks